docs/eventing/channel-based-broker.md

```diff
@@ -151,6 +151,25 @@ kubectl -n knative-eventing create rolebinding eventing-config-reader-default-ev
   --serviceaccount=default:eventing-broker-filter
```

Comment:
Thank you for updating doc related on configmappropagation! @liu-cong 
The paragraph from "Create RBAC permissions granting access to shared configmaps for logging,
tracing, and metrics configuration." to this end could be removed. The functionality of those two RBACs are replaced by the configmapprapagation.

Reply:
Thanks, done.